src/styles/global.css

```
@@ -2,6 +2,8 @@
 
 @import 'tailwindcss';
 
+@custom-variant hover (&:hover); /* Override Tailwind v4's @media (hover: hover) wrapper — fixes Chromium bug on touch-capable devices */
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)